### PR TITLE
PRO-1176: PLR Staking QA Feedback

### PR DESCRIPTION
**Pull request by Yash-Chaubal**
https://linear.app/pillarproject/issue/PRO-1176/pillar-wallet-plr-staking

## Description

- Fixed PLR asset select being disabled.
- Changed how some fees are displayed for swaps and bridges.
- Fixed crash for Archanova.

### CODE REVIEW

File: src/screens/PlrStaking/PlrStakingValidator.tsx [hunk 1]
```diff
@@ -201,13 +200,31 @@ const PlrStakingValidator = () => {
     if (!!asset) setPlrToken(asset);
   }, [toOptions]);
 
+  // Get Staking Fees
+  const { feeInfo: stakingFeeInfo } = useTransactionsEstimate(CHAIN.ETHEREUM, stakingTransactions, true, null);
+  if (stakingFeeInfo) console.log('stakingFeeInfo', stakingFeeInfo);
```
Comment (IAmKio) [inline src/screens/PlrStaking/PlrStakingValidator.tsx:205]: Sneaky console log here

File: src/screens/PlrStaking/PlrStakingValidator.tsx [hunk 2]
```diff
@@ -201,13 +200,31 @@ const PlrStakingValidator = () => {
...
       try {
-        const { transactions, stakeTransactions } = await buildStakeTransactions();
-        const allTransactions = isBridgeTransaction ? [...stakeTransactions] : [...transactions, ...stakeTransactions];
-        await getFeeInfo(allTransactions);
+        setCalculatingGas(true);
+        const { stakeTransactions } = await buildStakeTransactions();
+        setStakingTransactions(stakeTransactions);
       } catch (e) {
+        setCalculatingGas(false);
+        console.log('gettingGasFeeError', e);
```
Comment (IAmKio) [inline src/screens/PlrStaking/PlrStakingValidator.tsx:227]: Just another one here

File: src/screens/PlrStaking/PlrStakingValidator.tsx [hunk 3]
```diff
@@ -225,9 +242,12 @@ const PlrStakingValidator = () => {
       setSendData(data);
     };
 
-    getFee();
+    console.log('refreshingData');
```
Comment (IAmKio) [inline src/screens/PlrStaking/PlrStakingValidator.tsx:245]: ☝️ i'll stop pointing out console.logs but please have a look through your PR

**Review by parag-chauhan — APPROVED** [review]
LGTM

File: src/screens/PlrStaking/StakeRouteCard.tsx [hunk 4]
```diff
@@ -61,73 +67,86 @@ const StakeRouteCard: FC<IStakeRouteCard> = ({
...
+    const valueInFiat = getBalanceInFiat(currency, etherValue, !!isEthereum ? ethRates : chainRates, address);
+    return valueInFiat;
+  };
 
   return (
-    <RouteWrapper>
-      {!!sendData && (
+    <>
+      <RouteWrapper>
+        {/* Included empty tags as Route card UI breaks when keypad is unfocused - your guess is as good as mine */}
```
Comment (IAmKio) [inline src/screens/PlrStaking/StakeRouteCard.tsx:93]: 😂

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just some cleanup but other than that LGTM

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**